src/app/shared/object-detail/my-dspace-result-detail-element/item-search-result/item-search-result-detail-element.component.ts

```diff
@@ -24,8 +21,8 @@ import { Context } from '../../../../core/shared/context.model';
 export class ItemSearchResultDetailElementComponent extends SearchResultDetailElementComponent<ItemSearchResult, Item> {
 
   /**
-   * Represent item's status
+   * Represent context
    */
-  public status = MyDspaceItemStatusType.ARCHIVED;
+  public childContext = Context.MyDSpaceArchived;
```

Comment:
Here and in all the other result components, I'd rename it to badgeContext so it is clearer what it's referring to. could you change it?

Reply:
Agreed. I like the suggestion of calling this `badgeContext`.